### ISSUE TITLE
Sdk Version

### ISSUE THREAD

**Issue by VisionR1** [woesss/JL-Mod#87]
Hi i lookning inside in the manifest of your apps, and i found the min say 30 and the max say 29, it shouldn't have been the reverse or is it like this for some reason?

<uses-permission android:minSdkVersion="30" 
<uses-permission android:maxSdkVersion="29

**Comment by woesss** [comment]
Yes, that's right - these are restrictions for the property in which they are set.
